### ISSUE TITLE
Extract calls to SettingServiceBean from .xhtml

### ISSUE THREAD

**Issue by scolapasta** [IQSS/dataverse#2656]
We have several calls to the settingsServiceBean in our xhtml, but since they are in rendered attributes they get called multiple times. this results in multiple, unneeded queries to the DB.

So we will add a SettingsWrapper that gets all the setting once per view, and store in in a Map.

**Comment by kcondon** [comment]
@scolapasta I will perf test this but what are the functional areas affected so I can regression test those too?

**Comment by kcondon** [comment]
Gustavo: No functionality. Just calls to setting service from header and template. Calls were for status message and for Google analytics.

**Comment by kcondon** [comment]
Perf test showed changes resulted in slightly slower times but very minimal (1443ms avg versus 1512avg for changed version). Query counts were down by a couple hundred from 1600. There are still a lot of calls made from java code and leonid will look into that.

**Comment by landreev** [comment]
Didn't have time to work on this at all. Should be straightforward - just haven't gotten to it yet.

**Comment by landreev** [comment]
See the ticket #2685 (already referenced, above) for more info. 
Both the dataverse and dataset pages (and their included components) were still making direct calls to SystemConfig, to obtain some options stored in the Setting table. I replaced those with calls to Gustavo's SettingsWrapper. I also made some of the java beans involved call the wrapper as well. 

Gustavo's earlier fix (he replaced all the direct calls to SettingsServiceBean in xhtml with calls to his wrapper) actually took care of most of the setting lookups that were generated by the dataverse page. 
I just got rid of about 100 more (for the prod. db homepage, on dvn-vm5; measured while logged in as admin), so the remaining total is now 19. The homepage (the 1500+ files dataset on vm5, logged in as admin) generates 17. In theory, it should be possible to further reduce these to 1 or 2 - but I don't think it's worth the effort at this point.
